setup.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+POSTGRES_USER="user"
+POSTGRES_DB="gallery"
+
+#===============================================================================
+
+log()  { echo -e "\x1b[1m[\x1b[93mLOG\x1b[0m\x1b[1m]\x1b[0m ${@}";  }
+info() { echo -e "\x1b[1m[\x1b[92mINFO\x1b[0m\x1b[1m]\x1b[0m ${@}"; }
+warn() { echo -e "\x1b[1m[\x1b[91mWARN\x1b[0m\x1b[1m]\x1b[0m ${@}"; }
+
+install_pip_deps(){
+    log "Installing project dependencies ..."
+    python3 -m pip install -r requirements.txt 1>/dev/null
+    info "Done !"
+}
+
+prepare_db(){
+    log "Configuring container db"
+    POSTGRES_PASSWORD="$(cat /dev/urandom | base64 | head -n3 | tr -d "\n" | sed 's!+!!g' | sed 's!/!!g' | cut -b -20)"
```

Comment:
You don't `export` this variable, but you use it later in `prepare_django`. Are you sure this will still work? I'm no expert in Unix tho...